### PR TITLE
Creacion de la pagina de error y de la pagina de en construccion

### PR DESCRIPTION
**Pull request by marperoli2**

### CODE REVIEW

File: studybuddies/src/app/components/error-page/error-page.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ErrorPageComponent } from './error-page.component';
+
+describe('ErrorPageComponent', () => {
```
Comment (sanmargom2) [inline studybuddies/src/app/components/error-page/error-page.component.spec.ts:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
describe("ErrorPageComponent", () => {
```

File: studybuddies/src/app/components/error-page/error-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (sanmargom2) [inline studybuddies/src/app/components/error-page/error-page.component.ts:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
import { Component, OnInit } from "@angular/core";
```

File: studybuddies/src/app/app.module.ts [hunk 3]
```diff
@@ -20,6 +22,10 @@ const routes: Routes = [
   { path: 'tutor/createClass', component: CreateClassTutorComponent },
   { path: 'tutor/classList', component: ClassListTutorComponent },
   { path: '', component: HomeComponent },
+  { path: 'errorPage', component: ErrorPageComponent },
```
Comment (sanmargom2) [inline studybuddies/src/app/app.module.ts:25]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
  { path: "errorPage", component: ErrorPageComponent },
```

File: studybuddies/src/app/components/construction-page/construction-page.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ConstructionPageComponent } from './construction-page.component';
+
+describe('ConstructionPageComponent', () => {
```
Comment (sanmargom2) [inline studybuddies/src/app/components/construction-page/construction-page.component.spec.ts:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
describe("ConstructionPageComponent", () => {
```

File: studybuddies/src/app/components/error-page/error-page.component.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+@Component({
+  selector: 'app-error-page',
+  templateUrl: './error-page.component.html',
```
Comment (sanmargom2) [inline studybuddies/src/app/components/error-page/error-page.component.ts:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
  templateUrl: "./error-page.component.html",
```

File: studybuddies/src/app/components/error-page/error-page.component.ts (hunk 2 above)
Comment (sanmargom2) [inline studybuddies/src/app/components/error-page/error-page.component.ts:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
import { Component, OnInit } from "@angular/core";
```

File: studybuddies/src/app/components/error-page/error-page.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ErrorPageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (sanmargom2) [inline studybuddies/src/app/components/error-page/error-page.component.spec.ts:22]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
  it("should create", () => {
```

**Review by alvdiaasc97 — APPROVED** [review]

**Pull request merged**